File: CSharpChess/chess/King.cs
using CSharpChess.board;

namespace CSharpChess.chess

{
    public class King : Piece
    {
        public King(Board bd, Color color) : base(bd, color)
        {
            
        }
        public override string ToString()
        {
            return "K";
        }
    }
}

File: ChessJavaConsole/src/com/console/chess/pieces/King.java
package com.console.chess.pieces;

import com.console.boardgame.Board;
import com.console.chess.ChessPiece;
import com.console.chess.Color;

public class King extends ChessPiece {

    public King(Board board, Color color) {
        super(board, color);
    }

    @Override
    public String toString() {
        return "K";
    }

    @Override
    public boolean[][] possibleMoves() {
        boolean [][] mat = new boolean[getBoard().getRows()][getBoard().getColumns()];
        return mat;
    }
}
